### PR TITLE
Add functionality to define RabbitMQ port

### PR DESCRIPTION
**Pull request by RyanPrice-King**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -82,10 +84,12 @@ RUN sed -i '/\[jcli\]/a bind=0.0.0.0' /etc/jasmin/jasmin.cfg \
 &&  sed -i '/\[amqp-broker\]/a username='"$RABBITMQ_USERNAME" /etc/jasmin/jasmin.cfg \
 &&  sed -i '/\[amqp-broker\]/a password='"$RABBITMQ_PASSWORD" /etc/jasmin/jasmin.cfg \
 &&  sed -i '/\[amqp-broker\]/a host='"$RABBITMQ_HOST" /etc/jasmin/jasmin.cfg \
+&&  sed -i '/\[amqp-broker\]/a port='"$RABBITMQ_PORT" /etc/jasmin/jasmin.cfg \
 &&  sed -i '/\[amqp-broker\]/a heartbeat=15' /etc/jasmin/jasmin.cfg \
 &&  sed -i "s/host = '127.0.0.1'/host = '$RABBITMQ_HOST'/g" /usr/local/lib/python2.7/dist-packages/jasmin/sms_logger.py \
 &&  sed -i "s/username = 'guest'/username = '$RABBITMQ_USERNAME'/g" /usr/local/lib/python2.7/dist-packages/jasmin/sms_logger.py \
 &&  sed -i "s/password = 'guest'/password = '$RABBITMQ_PASSWORD'/g" /usr/local/lib/python2.7/dist-packages/jasmin/sms_logger.py \
+&&  sed -i "s/port = 5672/port = '$RABBITMQ_PORT'/g" /usr/local/lib/python2.7/dist-packages/jasmin/sms_logger.py \
```
Comment (marcoleary) [inline docker/Dockerfile]: What makes you think that this needs quotes around it?

**Review by marcoleary — CHANGES_REQUESTED** [review]
Won't you also need to edit the Jasmin code so that it will connect to the right port?

File: docker/Dockerfile (hunk 1 above)
Comment (RyanPrice-King) [reply docker/Dockerfile]: Wont need to add to Jasmin code as it is updating jasmin.cfg that the code uses.

Comment (RyanPrice-King) [reply docker/Dockerfile]: I removed the quotes from that part.

**Review by marcoleary — APPROVED** [review]
If Jasmin config already allows the port to be variable then this is good 👌🏼

**Pull request merged**